src/components/calcite-date-month/calcite-date-month.tsx

```diff
@@ -301,4 +274,59 @@ export class CalciteDateMonth {
     }
     return days;
   }
+
+  /**
+   * Determine if the date is in between the start and end dates
+   */
+  private betweenSelectedRange(date: Date): boolean {
+    return this.startDate && this.endDate && date > this.startDate && date < this.endDate;
+  }
+
+  /**
+   * Determine if the date should be in selected state
+   */
+  private isSelected(date: Date): boolean {
+    return sameDate(date, this.selectedDate) || (this.startDate && sameDate(date, this.startDate));
+  }
+
+  /**
+   * Render calcite-date-day
+   */
+  private renderDateDay(
+    active: boolean,
+    day: number,
+    dir: string,
+    date: Date,
+    currentMonth?: boolean,
+    ref?: boolean
+  ) {
+    const props = {
+      active,
+      ...(currentMonth && { currentMonth: true }),
```

Comment:
did this not work as

```
active,
currentMonth, 
...
```

I _think_ stencil will conditionally add/remove boolean props if what you pass it evaluates to false, but I've never passed a spread object like this, so I may be way off.

Reply:
You are right.  This was overkill, I've updated it.